src/content/docs/iast/applications.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: New Relic IAST Applications
+tags:
+    - IAST
+    - New Relic IAST
+    - Applications
+metaDescription: Find out how xxxx.
+---
+
+import iastCoverageSummary from 'images/iast_screenshot-crop_coverage-summary.webp'
+
+import iastEnableIast from 'images/iast_screenshot-crop_enable-iast.webp'
+
+
+The applications page informs you about the total number of found vulnerabilites in your tested applications and the number of untested applications you have. You can always add untested applications to the New Relic IAST capability.
+
+## Details and remediation of an application [#remediation-app]
+
+Check out all the issues detected with our New Relic IAS testing on your applications.
+
+To see the details and the remediation instructions for an application:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Applications** on the left navigation pane.
+
+2. Select an application. You can order all the columns alphabetically.
+
+<img
+    title="Details of an application"
+    alt="Details of an application"
+    src={iastCoverageSummary}
+/>
+
+## Enable New Relic IAST on an untested application [#enable-iast]
+
+To enable IAST on an untested application from the summary page:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Applications** on the left navigation pane.
+
+2. Click on the **Untested applications** tab and select the application you want to enable New Relic IAST.
+
+3. In the **Enable IAST** window, follow the steps to update your application configuration so it can be tested for exploitable vulnerabilities.
+
+    <img
+        title="Enable IAST for your untested application"
+        alt="Enable IAST for your untested application"
+        src={iastEnableIast}
+    />
+
+    <figcaption>
+        Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Applications** on the left navigation pane. Click on the **Untested applications** tab and select an application.
+    </figcaption>
+
+
+4. Update the APM agent if it's needed.
```

Comment:
How do I do this? How do I know it's needed?

Reply:
I'll ask Krystle.